### ISSUE TITLE
Incorrect parsing when adding terms to operator's namespace

### ISSUE THREAD

**Issue by asoltysik** [unisonweb/unison#2041]
```
(+*) : Text -> Text -> Nat
a +* b = Text.size (a ++ b)

test> (+*).test.ex1 = check (1 == 1)
```

Result
```
  I found a value  of type:  .test.ex1 ->{𝕖} [Result]
  where I expected to find:  [Result]
  
      1 | test> (+*).test.ex1 = check (1 == 1)
```

Looks like parser interprets `.test.ex1` as a paremeter. Not sure if this expected or not, but I can't find any other way to add tests/docs to operator's namespace.

**Codebase**: v2
**unison version**: devel/M1l-1388-g04040b6ff

**Comment by pchiusano** [comment]
Yeah this is a known issue though I'm on my phone and can't find the ticket. 😐 

The workaround is giving the function a regular name, like Text.appendSize, then do alias.term Text.appendSize Text.*+

**Comment by asoltysik** [comment]
Oh, sorry for the duplicate, didn't manage to find a similar ticket. Thanks for the workaround though!
